.github/workflows/tagged-release.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: Tagged Release
+on:
+  push:
+    tags:
+      - '7.0.[0-9]*'
+      - '7.0.[0-9]*-RC[0-9]*'
+      - '8.0.0-M[0-9]*'
```

Comment:
what does the 'M' mean here.

Reply:
Milestone. Nothing in the 8.0 branch is ready for production yet so they're all "milestone" releases.

Mostly just putting it here to save some hassle in the future when releases are actually created.